docs/content/documentation/systems/Postprocessors/phase_field/PolycrystalEBSD.md

```diff
@@ -1,6 +1,10 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # PolycrystalEBSD
+
+PolycrystalEBSD uses an EBSDReader object and utilizes the algorithms in [Polycrystal Initial Conditions](/ICs/PolycrystalICs.md) to generate
+a reduced order parameter model from electron backscatter data. More information about this capability can be found [here]((ICs/EBSD.md).
```

Comment:
I don't like using "here", also you seem to have an extra (.

 I would prefer something like:
```
...this capability is provided in the [EBSD](ICs/EBSD.md) object description.
```